src/components/pages/profile/profile.component.js

```diff
@@ -18,10 +19,15 @@ const Profile = ({ userData, fetchUserData }) => {
         title="Name"
         subTitle={userData.fullName}
       >
-        <h2>Description:</h2>
-
-        <ProfileDescriptionList userData={userData} />
-
+        <Grid container spacing={3}>
+          <Grid item xs={9}>
+            <h3>Details:</h3>
+            <ProfileDescriptionList userData={userData} />
+          </Grid>
+          <Grid item xs={1}>
+            <EditProfile userData={userData} />
+          </Grid>
+        </Grid>
```

Comment:
The grid system in material ui uses 12 blocks to operate. Is thin intentional here you are using only 10? I suggest to expand one of the xs items.

Reply:
It was a mistake. I wanted to use all the blocks